androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/playerOverlay/PlayerIcons.kt

```diff
@@ -95,3 +97,21 @@ fun CheckIcon(modifier: Modifier = Modifier) {
         modifier = modifier
     )
 }
+
+@Composable
+fun ForwardIcon(modifier: Modifier = Modifier) {
+    Image(
+        painter = painterResource(id = R.drawable.ic_forward),
+        contentDescription = stringResource(R.string.controls_forward_description),
+        modifier = Modifier.size(40.dp).testTag("ForwardIcon").then(modifier)
```

Comment:
nit: I was defining the size directly in the XML so what do I think to do the same to follow the other icons?